io.kapsa.drive.json

```diff
@@ -0,0 +1,338 @@
+          "tag": "0.9.1"
+        }
+      ]
+    },
+    {
+      "name": "libayatana-indicator",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/AyatanaIndicators/libayatana-indicator.git",
+          "tag": "0.9.1"
+        }
+      ]
+    },
+    {
+      "name": "libayatana-appindicator",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DENABLE_BINDINGS_MONO=NO",
+        "-DENABLE_BINDINGS_VALA=NO"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/AyatanaIndicators/libayatana-appindicator.git",
+          "tag": "0.5.90"
+        }
+      ]
+    },
+    {
+      "name": "libjson",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.9.5.zip",
+          "sha256": "a074e1b38083484e8e07789fd683599d19da8bb960959c83751cd0284bdf2043"
+        }
+      ]
+    },
+    {
+      "name": "libsecret",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dmanpage=false",
+        "-Dvapi=false",
+        "-Dgtk_doc=false",
+        "-Dintrospection=false"
+      ],
+      "cleanup": [
+        "/bin",
+        "/include",
+        "/lib/pkgconfig",
+        "/share/man"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libsecret/0.20/libsecret-0.20.5.tar.xz",
+          "sha256": "3fb3ce340fcd7db54d87c893e69bfc2b1f6e4d4b279065ffe66dac9f0fd12b4d"
+        }
+      ]
+    },
+    {
+      "name": "libsodium",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/jedisct1/libsodium/archive/1.0.18-RELEASE.tar.gz",
+          "sha256": "b7292dd1da67a049c8e78415cd498ec138d194cfdb302e716b08d26b80fecc10",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 1728,
+            "url-template": "https://github.com/jedisct1/libsodium/archive/$version.tar.gz"
+          }
+        }
+      ],
+      "post-install": [
+        "install -Dm644 LICENSE ${FLATPAK_DEST}/share/licenses/libsodium/LICENSE"
+      ]
+    },
+    {
+      "name": "s3drive",
+      "buildsystem": "simple",
+      "only-arches": [
+        "x86_64"
+      ],
+      "build-commands": [
+        "mkdir -p S3Drive",
+        "tar -xf S3Drive-Linux-Portable.tar.gz -C S3Drive",
+        "cp -r S3Drive /app/",
+        "chmod +x /app/S3Drive/kapsa",
+        "mkdir -p /app/bin",
+        "ln -s /app/S3Drive/kapsa /app/bin/kapsa",
+        "mkdir -p /app/share/metainfo",
+        "cp -r io.kapsa.drive.metainfo.xml /app/share/metainfo/",
+        "mkdir -p /app/share/icons/hicolor/scalable/apps",
+        "cp -r icon_flatpak.png /app/share/icons/hicolor/scalable/apps/io.kapsa.drive.png",
```

Comment:
since when a PNG is scalable?

Reply:
It's been replaced with SVG: https://github.com/s3drive/flathub/blob/s3drive/io.kapsa.drive.json#L310

Reply:
This is a lie:
```
icon_flatpak.svg: PNG image data, 512 x 512, 8-bit/color RGBA, non-interlaced
```